applications/ParticleMechanicsApplication/python_scripts/mpm_explicit_solver.py

```diff
@@ -25,7 +25,8 @@ def GetDefaultSettings(cls):
         this_defaults = KratosMultiphysics.Parameters("""{
             "time_integration_method"   : "explicit",
             "scheme_type"   : "central_difference",
-            "stress_update" : "usf"
+            "stress_update" : "usf",
+            "is_fix_explicit_mp_on_grid_edge" : false
```

Comment:
This sounds like it can/should be done with a process

Reply:
This is just adding the option to disable or enable the explicit MP on edge issue. Although it may be 'processy', I think it's better to keep it in the search utility because this is exactly where the problem is detected, and the fix for the problem requires additional searches (and can use the search structure already in scope) .